hermes/apps/athena/modules/db/src/stub.rs

```diff
@@ -0,0 +1,89 @@
+use crate::{hermes, DbComponent};
+
+impl hermes::exports::hermes::integration_test::event::Guest for DbComponent {
+    fn test(
+        _test: u32,
+        _run: bool,
+    ) -> Option<hermes::exports::hermes::integration_test::event::TestResult> {
+        None
+    }
+
+    fn bench(
+        _test: u32,
+        _run: bool,
+    ) -> Option<hermes::exports::hermes::integration_test::event::TestResult> {
+        None
+    }
+}
+
+impl hermes::exports::hermes::cardano::event_on_immutable_roll_forward::Guest for DbComponent {
+    fn on_cardano_immutable_roll_forward(
+        _subscription_id: &hermes::exports::hermes::cardano::event_on_immutable_roll_forward::SubscriptionId,
+        _block: &hermes::exports::hermes::cardano::event_on_immutable_roll_forward::Block,
+    ) {
+    }
+}
+
+impl hermes::exports::hermes::cardano::event_on_block::Guest for DbComponent {
+    fn on_cardano_block(
+        _subscription_id: &hermes::exports::hermes::cardano::event_on_block::SubscriptionId,
```

Comment:
Don't implement every single possible entry point.  You should only define endpoints that are active.  Hermes should use what you define to determine what events the module needs,  if they are all implemented it is not able to do that.

Reply:
The change is implemented, but there are currently two separate `worlds` living in the same `hermes` directory, which is problematic.

Additionally, the modules should be self contained, so the `db.wit` file needs to be moved into the `apps` directory. But for this to work, our build system must be modified. Hence this branch will be on-hold for some time (`no-track`) and additional GH issue will be created.